recipe/meta.yaml

```diff
@@ -68,9 +67,141 @@ about:
   home: http://askbot.org
   summary: Question and Answer forum, like StackOverflow, written in python and Django
   doc_url: https://pythonhosted.org/askbot/
-  license: BSD-3-Clause
-  license_file: LICENSE
+  license: MIT AND LGPL-3.0-or-later AND BSD-3-Clause AND GPL-3.0-or-later AND Apache-2.0 AND BSD-2-Clause
+  license_file:
+    - LICENSE
+    - COPYING
+    - askbot/conf/license.py
+    - askbot/media/node_modules/strip-json-comments/license
```

Comment:
For maintainability, have a look at other uses of `pnpm-licenses` in other feedstocks. This aggregates all licenses into one file.

Reply:
@xhochy - this is interesting, but might be beyond my capabilities - could you help here to make this use pnpm-licenses to aggregate the licenses..